Course/Course/Presentation/Dates/CourseDatesViewModel.swift

```diff
@@ -15,7 +15,20 @@ public class CourseDatesViewModel: ObservableObject {
     @Published var showError: Bool = false
     @Published var courseDates: CourseDates?
     @Published var dueDatesShifted: Bool = false
+    @Published var isOn: Bool = false
+    @Published var calendarEventsAdded: Bool = false
+    @Published var calendarEventsRemoved: Bool = false
+    @Published var calendarEventsUpdated: Bool = false
```

Comment:
Instead of using a boolean for each condition, how about using an enum for these cases?